src/main/kotlin/no/nav/syfo/aktivitetskrav/api/AktivitetskravApi.kt

```diff
@@ -47,7 +46,7 @@ fun Route.registerAktivitetskravApi(
                     val varsel = aktivitetskravVarselService.getVarsel(vurdering.uuid)
                     vurdering.toVurderingResponseDto(varsel)
                 }
-                aktivitetskrav.toResponseDTO(vurderingResponseDTOs)
+                AktivitetskravResponseDTO.from(aktivitetskrav, vurderingResponseDTOs)
```

Comment:
Hvis det er enkelt så kunne vi gjort det samme for responsen på POSTen som oppretter aktivitetskrav også.

Reply:
God idé! Se [kommentar](https://github.com/navikt/isaktivitetskrav/pull/165/files#r1389432141)